Source/Chronozoom.Entities/Chronozoom.Entities.csproj

```diff
@@ -171,6 +171,7 @@
     <Content Include="SQL\ManualMigration\201408130000000_MultipleCollections.sql" />
     <Content Include="SQL\ManualMigration\201409250000000_PubliclySearchableChange.sql" />
     <Content Include="SQL\ManualMigration\201502100000000_AddBackgroundUrlAspectRatio.sql" />
+    <Content Include="SQL\ManualMigration\201502170000000_AddYOffsetsAttributes.sql" />

```

Comment:
It were only in my own fork. I removed it before pull request creation and transferred the code to AddYOffsetsAttributes.sql. Is there any problem?